eZ/Publish/Core/Limitation/BlockingLimitationType.php

```diff
@@ -109,7 +109,7 @@ public function buildValue(array $limitationValues)
      * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException If any of the arguments are invalid
      *         Example: If LimitationValue is instance of ContentTypeLimitationValue, and Type is SectionLimitationType.
      * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException If value of the LimitationValue is unsupported
-     *         Example if OwnerLimitationValue->limitationValues[0] is not one of: [ 1,  2 ]
+     *         Example if OwnerLimitationValue->limitationValues[0] is not one of: [ 1,  2 ]
```

Comment:
Matched by `non_printable_character`. All of the changes are related to some whitespace characters outside of standard ASCII table.